.github/workflows/ci-unix-static.yml

```diff
@@ -78,6 +78,7 @@ jobs:
       - name: Check static link bundling
         run: |
           ${{ env.CC }} -o avifenc  -I./apps/shared -I./third_party/iccjpeg -I./include apps/avifenc.c \
+            -I/opt/homebrew/include/ -L/opt/homebrew/lib \
```

Comment:
One way to avoid this is to build a simple test program instead of avifenc.c. Then we don't need to compile `apps/shared/*.c third_party/iccjpeg/iccjpeg.c` and link with -lpng -ljpeg -lz.